scalalib/src/PublishModule.scala

```diff
@@ -18,6 +18,11 @@ trait PublishModule extends JavaModule { outer =>
   def pomSettings: T[PomSettings]
   def publishVersion: T[String]
 
+  /**
+   * Version scheme used for the model: Supported values are VersionScheme.EarlySemVer, VersionScheme.PVP, and VersionScheme.SemVerSpec
+   */
+  def versionScheme: T[VersionScheme] = VersionScheme.NoScheme
```

Comment:
What do you think of making it an `Option[VersionScheme]` instead? (and modeling `NoScheme` as `None`, and getting rid of the inner `Option` in `VersionScheme`?

(Not sure this is important!)